keyboards/ergotaco/readme.md

```diff
@@ -0,0 +1,22 @@
+# Gergo
+
+![ErgoTaco](https://i.redd.it/dbcu5i21m3i21.jpg)
+
+It's a Taco with a side of HASL. A 12 key fiesta of a board.
+
+Keyboard Maintainer: [Jeremy Bernhardt](https://github.com/germ)  
+Hardware Supported: ErgoTaco
+Hardware Availability: [gboards.ca](http://gboards.ca)  
+
+## Firmware building
+After cloning the QMK repo and installing dfu-programmer build and flash with. Be sure to reset your keyboard!
```

Comment:
I think Lines 12-14 would be better as:

```
After cloning the QMK repo and installing dfu-programmer, build and flash with:

    make ergotaco:default:dfu

Be sure to reset your keyboard!

Reply:
The first sentence here still bugs me, with its improper grammar.

\- @noroadsleft 
*son of an English major*